guide/english/certifications/javascript-algorithms-and-data-structures/basic-algorithm-scripting/repeat-a-string-repeat-a-string/index.md

```diff
@@ -65,12 +65,13 @@ Make the variable created store the current value and append the word to it.
 ## ![:sunflower:](https://forum.freecodecamp.com/images/emoji/emoji_one/sunflower.png?v=3 ":sunflower:") Intermediate Code Solution:
 
     function repeatStringNumTimes(str, num) {
-      if(num < 0)
+      if (num < 1) {
         return "";
-      if(num === 1)
+      } if (num === 1) {
```

Comment:
Though it technically will not change the result, if you are going to write it with the curly brackets, let's make it an `else if ` here:
```suggestion
      } else if (num === 1) {
```